components/TestLibraryStepper/TestLibraryStepper.module.scss

```diff
@@ -0,0 +1,19 @@
+﻿@import '../../pageStyles/constants';
+
+.stepperBtn {
+  width: 135px;
+  height: 30px;
+  margin: 30px;
+  text-decoration: none;
+  align-items: center;
+  text-align: center;
+  border-width: 9px;
+  border-color: $softwireYellow;
+  border-style: solid;
+  background-color: $softwireYellow;
+  color: $softwireBlack;
+  font-size: $buttonFontSize;
+  justify-content: center;
```

Comment:
I think `justify-content` and `align-items` will only work if its `display: flex`, which it doesn't look like it is.
Can we just remove them?

Reply:
done. ✔️